packages/predictions/src/Providers/AmazonAIConvertPredictionsProvider.ts

```diff
@@ -102,29 +110,26 @@ export class AmazonAIConvertPredictionsProvider extends AbstractConvertPredictio
 		input: TextToSpeechInput
 	): Promise<TextToSpeechOutput> {
 		const { credentials } = await fetchAuthSession();
-		if (!credentials) {
-			return Promise.reject('No credentials');
-		}
-		const {
-			speechGenerator: {
-				defaults: { voiceId: defaultVoiceId = '' } = {},
-				region = '',
-			} = {},
-		} = Amplify.getConfig().Predictions?.convert || {};
-
-		if (!input.textToSpeech.source) {
-			return Promise.reject('Source needs to be provided in the input');
-		}
-		const voiceId = input.textToSpeech.voiceId || defaultVoiceId;
-		if (!region) {
-			return Promise.reject(
-				'Region was undefined. Did you enable speech generator using amplify CLI?'
-			);
-		}
+		assertValidationError(
+			!!credentials,
+			PredictionsValidationErrorCode.NoCredentials
+		);
+		assertValidationError(
+			!!input.textToSpeech.source,
```

Comment:
Nit to make it more fault tolerant for runtime validation

```suggestion
			!!input.textToSpeech?.source,
```